src/Engines/thinEngine.ts

```diff
@@ -3556,16 +3556,19 @@ export class ThinEngine {
 
                 prepareTexture(texture, extension, scene, img, texture.invertY, noMipmap, false, prepareTextureProcessFunction, samplingMode);
             };
+            // According to the WebGL spec section 6.10, ImageBitmaps must be inverted on creation.
+            // So, we pass imageOrientation to _FileToolsLoadImage() as it may create an ImageBitmap.
+            let imageBitmapOptions: ImageBitmapOptions = { imageOrientation: texture.invertY ? "flipY" : "none" };

```

Comment:
Could we move the option creation near the _FileToolsLoadImage to prevent unnecessary object creation then GC when not needed ?